package/native-package/package.json

```diff
@@ -56,7 +56,7 @@
   "devDependencies": {
     "@bam.tech/react-native-image-resizer": "^3.0.10",
     "@react-native-camera-roll/camera-roll": "^5.0.2",
-    "react-native": ">=0.73.6",
+    "react-native": "^0.73.6",
```

Comment:
Check the comment in `yarn.lock`

Reply:
Yes, it's good for now, until we test with 0.74 on native CLI. I can do that once I move to testing new arch with native cli